.gitignore

```diff
@@ -0,0 +1,15 @@
+# Compiled files
+*.o
+*.so
+*.rlib
+*.dll
+
+# Executables
+*.exe
+
+# Generated by Cargo
+/target/
+
+# Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
+# More information here http://doc.crates.io/guide.html#cargotoml-vs-cargolock
+Cargo.lock
```

Comment:
I'd like to avoid cargo cults where possible. `cargo` places all binaries in `/target` making that the necessary and sufficient entry to handle all of them.